openid-connect-server/src/test/java/org/mitre/oauth2/service/impl/TestDefaultSystemScopeService.java

```diff
@@ -184,6 +184,60 @@ public void scopesMatch() {
 
 		// extra scope (fail)
 		assertThat(service.scopesMatch(expected, actualBad), is(false));
+		
 	}
+	
+	@Test
+	public void getremoveRestrictedAndReservedScopes() {
+						
+		Set<SystemScope> restrictedAndReservedScopes = Sets.newHashSet(defaultDynScope1, defaultDynScope2, dynScope1);
+		assertThat(service.removeRestrictedAndReservedScopes(allScopes),equalTo(restrictedAndReservedScopes));
+	}
+	
+	@Test
+	public void getremoveReservedScopes() {
+		Set<SystemScope> removeReservedScopes = Sets.newHashSet(defaultDynScope2, defaultDynScope1, defaultScope2,restrictedScope1,defaultScope1,dynScope1);
+		assertThat(service.removeReservedScopes(allScopes), equalTo(removeReservedScopes));
+	}
+	
+	@Test 
+	public void getById_notfound() {
+		
+		// check null condition
+
+		assertThat(service.getById((long) 60), is(nullValue()));
+
+		
+	}
+	
+	@Test 
+	public void getByValue_notfound() {
+		
+		// check null condition
+		
+		assertThat(service.getByValue("defaultDynScope1String"), is(nullValue()));		
+		
+		
+	}
+	
+	@Test
+	public void save_scope() {
+		
+		// check null condition 
+
+		assertThat(service.save(defaultScope1), is(nullValue()));
+		assertThat(service.save(mySystemScope), equalTo(mySystemScope));
+
+
+		
+	}
+	
+	@Test
+	public void getReserveScopes() {
+		Set<SystemScope> reserved = SystemScopeService.reservedScopes;
```

Comment:
No need to pull this out as a separate variable, just reference it in the test below